src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -1,9 +1,93 @@
 package frc.robot.maps;
 
+import com.chopshop166.chopshoplib.drive.MockSwerveModule;
+import com.chopshop166.chopshoplib.drive.SwerveModule;
 import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.sensors.MockGyro;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 
 // Need to get MAC address for roborio
 @RobotMapFor("Default")
 public class RobotMap {
+    public static class DriveMap {
+        // All Distances are in Meters
+
+        private final SwerveModule frontLeft;
+        private final SwerveModule frontRight;
+        private final SwerveModule rearLeft;
+        private final SwerveModule rearRight;
+        private final double maxDriveSpeedMetersPerSecond;
+        private final double maxRotationRadianPerSecond;
+        private final Gyro gyro;
+
+        public DriveMap() {
+
+            this.frontLeft = new MockSwerveModule(new Translation2d(0.381, 0.381));
```

Comment:
Maybe make `0.381` a constant? where did this number come from? What unit is it in? (Yes they're mocks, but it's still good practice)